homeassistant/components/frontier_silicon/config_flow.py

```diff
@@ -37,12 +37,8 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    def __init__(self) -> None:
-        """Initialize flow."""
-
-        self._webfsapi_url: str | None = None
-        self._name: str | None = None
-        self._unique_id: str | None = None
+    _webfsapi_url: str | None = None
+    _unique_id: str | None = None
```

Comment:
I think `self._unique_id` isn't needed at all. It can always be changed to the local variable.